docs/contributorfile.rst

```diff
@@ -132,24 +146,23 @@ Issues labels
 
 .. _prlabel:
 
-PRs labels
-~~~~~~~~~~
--  BugFIX: These PRs close an issue labelled ``bug``. they also increase the semantic versioning for fixes (+0.0.1).
--  Invalid: These PRs don't seem right. They actually seem so not right that they won’t be further processed. This label invalidates an Hacktoberfest contribution. If you think this is wrong, start a discussion in the relevant issue (or open one if missing). Reviewers are asked to give an explanation for the use of this label.
+PRs only labels
+~~~~~~~~~~~~~~~
+
```

Comment:
Just a comment here. The spacing and weight of the titles feel a bit weird to me. I assume this is the result of the necessary break lines in rst.

Reply:
Yep, it is!